packages/server/api/controllers/reviews.controller.js

```diff
@@ -92,10 +92,33 @@ const deleteReview = async (ReviewID) => {
   return knex('reviews').where({ id: ReviewID }).del();
 };
 
+const addReview = async (movieId, userId, rating, reviewText) => {
+  if (isNaN(movieId)) {
+    throw new HttpError('Movie ID should be a number', 400);
+  }
+  if (isNaN(userId)) {
+    throw new HttpError('User ID should be a number', 400);
+  }
+
+  try {
+    const review = await knex('reviews').insert({
+      movie_id: movieId,
+      user_id: userId,
+      rating,
+      review_text: reviewText,
+      created_at: moment().format(),
+    });
+    return review;
```

Comment:
no need to return review if you are not going to use it :)

Reply:
ah, I see. Normally you don't return from inside of try clause, you return after the block.